packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFWebViewConfigurationHostApiTests.m

```diff
@@ -62,6 +62,26 @@ - (void)testSetAllowsInlineMediaPlayback {
   XCTAssertNil(error);
 }
 
+- (void)testsetLimitsNavigationsToAppBoundDomains {
+  if (@available(iOS 14, *)) {
```

Comment:
My preference is to use API_AVAILABLE(ios(14.0)) in the method declaration. See https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFUserContentControllerHostApiTests.m#L69